packager/media/formats/mp4/mp4_muxer.cc

```diff
@@ -206,10 +206,11 @@ Status MP4Muxer::DelayInitializeMuxer() {
   std::unique_ptr<FileType> ftyp(new FileType);
   std::unique_ptr<Movie> moov(new Movie);
 
-  ftyp->major_brand = FOURCC_isom;
+  ftyp->major_brand = FOURCC_mp41;
   ftyp->compatible_brands.push_back(FOURCC_iso8);
   ftyp->compatible_brands.push_back(FOURCC_mp41);
   ftyp->compatible_brands.push_back(FOURCC_dash);
+  ftyp->compatible_brands.push_back(FOURCC_avc1);
```

Comment:
I thought you removed a fourcc in the original code. Didn't notice that you hard-coded it yourself. It is already included in line 223.

Sorry for my oversight. 

Can you replace it with FOURCC_isom? And move it after iso8?